rotkehlchen/db/schema.py

```diff
@@ -632,13 +632,25 @@
     notes TEXT,
     type TEXT NOT NULL,
     subtype TEXT NOT NULL,
-    counterparty TEXT,
-    extra_data TEXT,
     FOREIGN KEY(asset) REFERENCES assets(identifier) ON UPDATE CASCADE,
     UNIQUE(event_identifier, sequence_index)
 );
 """
 
+
+# Table that extends history_events table and stores data specific to evm events.
+DB_CREATE_EVM_EVENTS_INFO = """
+CREATE TABLE IF NOT EXISTS evm_events_info(
+    identifier INTEGER PRIMARY KEY,
+    counterparty TEXT,
```

Comment:
Shouldn't this be protocol? It's really confusing that these two terms are used interchangably